lib/doorkeeper/rails/routes.rb

```diff
@@ -40,11 +40,19 @@ def generate_routes!(options)
           map_route(:applications, :application_routes)
           map_route(:authorized_applications, :authorized_applications_routes)
           map_route(:token_info, :token_info_routes)
+          device_flow_routes
```

Comment:
Could we please rename the method to `map_device_flow_routes` ? I think it more semantically fit here